traffic_ops/testing/api/v14/config/common.go

```diff
@@ -0,0 +1,87 @@
+/*
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"time"
+)
+
+// Defines the 24hr format
+// see https://golang.org/pkg/time/#Parse
+const militaryTimeFmt = "15:04"
+
+// Validate24HrTimeRange determines whether the provided
+// string fits in a format such as "08:00-16:00"
```

Comment:
Nitpick: Missing period. GoDoc comments should be full sentences, i.e. `// string fits in a format such as "08:00-16:00".`